core/forms/MemberForms.py

```diff
@@ -258,6 +267,8 @@ def clean_image(self):
         """Ensures that the image is of a sufficiently small size before it gets uploaded"""
         image = self.cleaned_data["image"]
 
+        if image.name == DEFAULT_IMG:
+            raise forms.ValidationError("Please upload your own profile picture!")
```

Comment:
I think members should be allowed to register without a photo and keep the shaka or something like @sarahwells12595 uses as default

Reply:
We do want them all to have photos, just so we can identify them. If they really don't want to show their face they can sill upload a random photo of a dog or whatnot. This is mostly to ensure they don't accidentally skip the step